scripts/us_bls/jolts/BLSJolts_StatisticalVariables.mcf

```diff
@@ -0,0 +1,1416 @@
+
```

Comment:
New schema from this import should go into [schema/bls.mcf](https://github.com/datacommonsorg/data/blob/master/schema/bls.mcf)

Reply:
Changed.